CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+# How to Contribute
+
+We'd love to accept your patches and contributions to this project.
+
+## Before you begin
+
+### Sign our Contributor License Agreement
+
+Contributions to this project must be accompanied by a
+[Contributor License Agreement][cla] (CLA). You (or your employer) retain the
+copyright to your contribution; this simply gives us permission to use and
+redistribute your contributions as part of the project.
+
+If you or your current employer have already signed the Google CLA (even if it
+was for a different project), you probably don't need to do it again.
+
+Visit <https://cla.developers.google.com/> to see your current agreements or to
+sign a new one.
+
+### Review our Community Guidelines
+
+This project follows [Google's Open Source Community
+Guidelines](https://opensource.google/conduct/).
+
+## Contribution process
+
+We follow the [fork and branch workflow][workflow].
+
+There will be three Git repositories involved:
+
+1.  *upstream* - the Service Weaver KUBE repository on GitHub.
```

Comment:
In a couple places, you say `Service Weaver KUBE`. What do you think about saying `weaver-kube` if we're referring to the repository, `Service Weaver Kubernetes deployer` if we're referring to the deployer abstractly, and `weaver kube` if we're referring to the actual `weaver kube` binary?

Reply:
Good point.